### ISSUE TITLE
Google search link to Ionicons links to v4 icons, yet current version is v5

### ISSUE THREAD

**Issue by pnyro** [ionic-team/ionicons#1052]
To reproduce:

1. Google "ionicons"
2. Click on the link that goes directly to the icons library (not the main link but the sub-link)
3. Goes to `https://ionic.io/ionicons/v4`

**Comment by BerkerYuceer** [comment]
Yes, issue is confirmed by me. Directs to [v4](https://ionic.io/ionicons/v4)

![Ekran Alıntısı](https://user-images.githubusercontent.com/2599466/158337354-70457a5e-81d0-437b-ab4e-946dca11aad7.PNG)

**Comment by christian-bromann** [comment]
This issue has been resolved.
